### ISSUE TITLE
Implement output post processing capabilities

### ISSUE THREAD

**Issue by chahuistle** [genericworkflownodes/GenericKnimeNodes#45]
EMBOSS offers a mature toolset, but for some reason, the output of graphs does not work as expected. Following is an example:

$ chaos -sequence foo.fasta -graph ps -goutfile foo.ps

Here you are telling the tool chaos that you want to generate a graph in ps format in the foo.ps file. The input file is foo.fasta. What happens is that the output gets generated in foo.ps.ps, so the desired format is appended to the file name.

GKN should be able to overcome this via a property in plugin.properties.

**Comment by aiche** [comment]
As this is more a tool then a plugin related issue I would suggest moving a corresponding property to the CTD files. Currently I have something like 

```
<output-converters>
  <converter class="FileNameExtender" ref="emboss.graph">
    <converter:property name="extension" value=".ps" />
  </converter>
</output-converters>
```

in mind. The class attribute would name a class that can be instantiated at run-time (maybe using eclipse's extension point mechanism), the ref attribute would point to an item inside the ITEM part of the CTD, and the properties part can be used to configure the converter.

**Comment by chahuistle** [comment]
It's a great idea. We can use "FileNameExtender" or "FileNameReplacer" or whatever is needed to actually find the file.

**Comment by aiche** [comment]
fixed in 5ead86aed5980371d8cdcbd1d42eaa6b7b637caa
